src/main/java/com/wcc/platform/controller/CmsController.java

```diff
@@ -0,0 +1,26 @@
+package com.wcc.platform.controller;
+
+import com.wcc.platform.domain.pages.TeamPage;
+import com.wcc.platform.service.CmsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/cms/v1/")
```

Comment:
I think because of costs now, we will have in a single service, but that was my original plan, to have one microservice for cms backend and another one for platform. But for know, and because of costs, let's have a single microservice and just split the services, for example now, I have one for CMS, but we can split more if we need.